polaris-react/src/components/ProgressBar/tests/ProgressBar.test.tsx

```diff
@@ -24,20 +25,34 @@ describe('<ProgressBar />', () => {
     expect(progress).toContainReactComponent('progress', {value: 0});
   });
 
+  it('renders a CSSTransition component', () => {
+    const progress = mountWithApp(<ProgressBar />);
+
+    expect(progress).toContainReactComponent(CSSTransition, {
+      in: true,
+      appear: true,
+      timeout: 500,
+      classNames: {
+        appearActive: 'IndicatorAppearActive',
+        appearDone: 'IndicatorAppearDone',
+      },
+    });
+  });
+
   describe('animated prop', () => {
-    it('sets the progress bar to include the Animated class by default', () => {
+    it('sets the progress bar CSSTransition to have a non-zero timeout value by default', () => {
       const progress = mountWithApp(<ProgressBar progress={20} />);
-      expect(progress).toContainReactComponent('div', {
-        className: 'Indicator Animated',
+      expect(progress).toContainReactComponent(CSSTransition, {
+        timeout: 500,
```

Comment:
I was hoping to just make this value any value greater than zero but struggled to find the right syntax.